server/komodo-rest/src/main/java/org/komodo/servicecatalog/datasources/AmazonS3Definition.java

```diff
@@ -29,7 +30,7 @@
 
     @Override
     public String getType() {
-        return "amazon-s3";
+        return "webservice";
```

Comment:
Why is this now more generic given its in the named class, please?

Reply:
This represents the resource-adapter internally.